Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -385,6 +385,16 @@ class Gx2Fitter {
                     const Logger& /*logger*/) const {
       assert(result.fittedStates && "No MultiTrajectory set");
 
+      // Add the measurement surface as external surface to the navigator.
+      // We will try to hit those surface by ignoring boundary checks.
+      if (state.navigation.externalSurfaces.size() == 0) {
+        for (auto measurementIt = inputMeasurements->begin();
+             measurementIt != inputMeasurements->end(); measurementIt++) {
+          navigator.insertExternalSurface(state.navigation,
+                                          measurementIt->first);
+        }
+      }
```

Comment:
Orthogonal to this PR but I think we should really put this as a configuration on the navigator

Reply:
Let's also ping this issue ;)
- https://github.com/acts-project/acts/issues/2861